src/pages/admin/reports/components/funds-expenditures-section/FundsExpendituresSection.module.scss

```
@@ -0,0 +1,100 @@
+    background-color: c.$white;
+    border: 1px solid c.$grey-500;
+    border-radius: 16px;
+    box-shadow: 0 11px 18px rgba(c.$bluish-black, 0.19);
+    display: flex;
+    flex-direction: column;
+    gap: 20px;
+}
+
+.disclaimer-label {
+    @include type.subtitle-1-medium;
+    line-height: 24px;
```

Stylelint (17.4.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)